### PR TITLE
Tal/flippy tiles

### PR DESCRIPTION
**Pull request by talsraviv**
Documenting changes reflected in https://github.com/duckduckgo/zeroclickinfo-spice/pull/1981 and corresponding internal changes

### CODE REVIEW

**Comment by talsraviv** [comment]
@moollaza take a look - make sure I got it right this time.

File: duckduckhack/instant-answer-display/templates_reference.md [hunk 1]
```diff
@@ -652,6 +653,199 @@ This view is displayed when the 'front' is clicked, together with the 'back' (ab
...
+### Available Features
+
+#### 'Front' of each item:
+
+These properties are passed to the template inside a `data_front` object (e.g. `data_front.title`).
+
+- `icon` [optional]
+- `title` [optional]
+- `altsubtitle` [optional]
+- `aubtitle` [optional]
```
Comment (moollaza) [inline duckduckhack/instant-answer-display/templates_reference.md]: small typo

File: duckduckhack/instant-answer-display/templates_reference.md [hunk 2]
```diff
@@ -652,6 +653,199 @@ This view is displayed when the 'front' is clicked, together with the 'back' (ab
...
+- `footer_content` [optional] [*sub-template*](https://duck.co/duckduckhack/subtemplates)
+
+#### 'Back' of each item: (displayed upon click)
+
+These properties are passed to the template inside a `data_back` object (e.g. `data_back.title`).
+
+- `icon` [optional]
+- `title` [optional]
+- `altsubtitle` [optional]
+- `aubtitle` [optional]
```
Comment (moollaza) [inline duckduckhack/instant-answer-display/templates_reference.md]: same here :)

**Comment by moollaza** [comment]
@talsraviv aside from the two typos, LGTM :+1:

**Comment by talsraviv** [comment]
Fixed, merging

**Pull request merged**